pmaweb/templates/_sponsorship_conditions.html

```diff
@@ -2,6 +2,7 @@ <h3 id="rules">Sponsorship conditions and advantages</h3>
 
 <ul>
  <li>A sponsor has to commit to a yearly amount depending on the sponsorship level</li>
+ <li>Sponsorships are always on an annual basis and are NOT refundable once paid</li>
```

Comment:
Maybe using HTML markup instead of caps is nicer?

```suggestion
 <li>Sponsorships are always on an annual basis and are <strong>not</strong> refundable once paid</li>
```

Reply:
Thanks, I've modified this to include your suggestion.